config/sync/system.site.yml

```diff
@@ -12,3 +12,4 @@ langcode: en
 default_langcode: en
 _core:
   default_config_hash: yTxtFqBHnEWxQswuWvkjE8mKw2t8oKuCL1q8KnfHuGE
+mail_notification: ''
```

Comment:
I thought i made a comment on this already... but what is this for?

Reply:
you did. It happened when I enabled email verification.

Reply:
Okay cool thanks.